api/app/db/crud/api_access_audits.py

```diff
@@ -0,0 +1,28 @@
+""" CRUD create function for audit logs
+"""
+from datetime import timezone, datetime
+import logging
+from sqlalchemy.exc import IntegrityError
+from app.db.models import APIAccessAudit
+
+import app.db.database
+
+logger = logging.getLogger(__name__)
+
+
+def create_api_access_audit_log(
+        username: str,
+        success: bool,
+        path: str) -> APIAccessAudit:
+    """ Create an audit log. """
+    with app.db.database.get_write_session_scope() as session:
+        now = datetime.now(tz=timezone.utc)
```

Comment:
just an f.y.i - there's a app.time_utils.get_utc_now that does exactly this, for ensuring that unit tests give consistent results it's mocked out by default in api/app/routers/forecasts.py ;
Doesn't matter in this context, and there isn't a unit test looking at the date - but just so you know.